gui/src/ts/components/node/NodePredicateGrid.tsx

```diff
@@ -0,0 +1,60 @@
+import {NodePageQuery_nodeById_subjectOfEdges} from "api/queries/types/NodePageQuery";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  List,
+  ListItem,
+  Grid,
+} from "@material-ui/core";
+import * as React from "react";
+import {NodeLink} from "./NodeLink";
+
+const EDGE_PREDICATE_DISPLAY_NAMES: {[index: string]: string} = {};
+
+const EdgeList: React.FunctionComponent<{
+  edges: NodePageQuery_nodeById_subjectOfEdges[];
+  predicate: string;
+}> = ({edges, predicate}) => {
+  let title = EDGE_PREDICATE_DISPLAY_NAMES[predicate];
+  if (!title) {
+    title = predicate;
+  }
+  return (
+    <Card>
+      <CardHeader
+        data-cy="edge-list-title"
+        title={title}
+        style={{textAlign: "center"}}
+      />
+      <CardContent>
+        <List>
+          {edges.map((edge) => (
+            <ListItem data-cy="edge" key={edge.object}>
+              <NodeLink node={edge.objectNode!} />
+            </ListItem>
+          ))}
+        </List>
+      </CardContent>
+    </Card>
+  );
+};
+
+export const NodePredicateGrid: React.FunctionComponent<{
+  predicateSubjects: {
+    [predicate: string]: NodePageQuery_nodeById_subjectOfEdges[];
+  };
+}> = ({predicateSubjects}) => {
+  return (
+    <Grid container spacing={4}>
+      {Object.keys(predicateSubjects).map((predicate) => (
```

Comment:
We have to do this idiom all over, and having to do obj[key]! sucks. There's a method in lodash, pairs, that you could also use. No need to change this.